lib/backend/compat/compat.go

```diff
@@ -144,5 +144,13 @@ func toTagsLabelsRules(d *KVPair) (t, l, r *KVPair) {
 		Value: p.Rules,
 	}
 
+	// Fix up tags and labels so to be empty values rather than nil.
```

Comment:
This is purely for Felix - we shouldn't be writing null where Felix would be expecting a dictionary or a list.